Kickstarter-iOS/Views/Controllers/FeatureFlagToolsViewController.swift

```diff
@@ -0,0 +1,109 @@
+import Foundation
+import KsApi
+import Library
+import Prelude
+import UIKit
+
+final class FeatureFlagToolsViewController: UITableViewController {
+  private let viewModel = FeatureFlagToolsViewModel()
```

Comment:
Is there a reason why are we not placing `VM` under properties? Maybe if this is something we should do we need to document it so that everyone embraces it.

Reply:
I think historically we've always but the view model at the top 🤷‍♀ I can move it to be part of the properties.